src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -128,20 +116,47 @@
         personalisation.put(ONLINE_HEARING_REGISTER_LINK_LITERAL, config.getOnlineHearingLink() + "/register");
         personalisation.put(ONLINE_HEARING_SIGN_IN_LINK_LITERAL, config.getOnlineHearingLink() + "/sign-in");
 
+        personalisation.put(APPOINTEE_DESCRIPTION, getAppointeeDescription(subscriptionType, ccdResponse));
+
         return personalisation;
     }
 
+    private String getName(SubscriptionType subscriptionType, SscsCaseData ccdResponse) {
+        Name name = null;
+        if (subscriptionType.equals(APPELLANT) && ccdResponse.getAppeal() != null
+                && ccdResponse.getAppeal().getAppellant() != null) {
+            name = ccdResponse.getAppeal().getAppellant().getName();
+        } else if (subscriptionType.equals(REPRESENTATIVE) && ccdResponse.getAppeal() != null
```

Comment:
why Reps? the ticket does not say a thing about Reps. Isn't that managed as part of another ticket?

Reply:
name is used in some existing templates and we have refactored whom the name is. As we now know who we are addressing the notification to, we should address the rep by the name instead of an empty string. i.e. Dear <empty string>, rest of content...